functions/get_postgres_hosts.pp

```diff
@@ -0,0 +1,18 @@
+function rsan::get_postgres_hosts() {
+  if $settings::storeconfigs {
```

Comment:
this makes this whole thing conditional on $settings::storeconfigs, which is a snippet from another place in the code. This does not need to be conditional